tsconfig.json

```diff
@@ -5,6 +5,7 @@
     "moduleResolution": "node",
     "esModuleInterop": true,
     "strictNullChecks": true,
+    "declaration": true,
```

Comment:
for my education, can you tell me what this line does?

Reply:
here is more about it https://www.typescriptlang.org/tsconfig/declaration.html